src/views/Alteration.vue

```diff
@@ -214,6 +214,7 @@ export default class Alteration extends Mixins(CommonMixin, FeeMixin, FilingTemp
     }
   }
 
+  @Watch('hasBusinessNameChanged')
```

Comment:
I don't know if this change is the reason why, but I found that I can Undo the Company Name and the Business Type remains changed.

![image](https://github.com/user-attachments/assets/139b2d61-76b6-4060-a302-1b0029559ca8)

Should this make the filing invalid?

Should the Business Type also be undone?

Reply:
Oh, wait.

When I saved and resumed, the resume was in the old Edit UI. Let me test this again.

Reply:
I just checked this on dev and this issue already exists but I can look into fixing this. Should I update the scope on this ticket?

Reply:
Please test this thoroughly. I believe it's working well.